### PR TITLE
JNG-4800 JSL model name with java keywords throw error on the psm-gen…

### PR DESCRIPTION
**Pull request by Madmat8**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4800" title="JNG-4800" target="_blank">BugJNG-4800</a>  JSL model name with java keywords throw error on the psm-generator
  </td></table>
  <br />
 

…erator

### CODE REVIEW

**Review by robertcsakany — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -39,9 +39,10 @@
             javax.transaction.xa;version=!,
             javax.resource;version="[1.6,2)"
         </osgi-transaction-import>
-        <judo-tatami-jsl-version>1.1.4.20230610_040712_819bbeeb_develop</judo-tatami-jsl-version>
-        <judo-runtime-core-version>1.0.6.20230609_081903_0735e4f8_develop</judo-runtime-core-version>
-        <judo-psm-generator-sdk-core-version>1.0.0.20230607_141556_5a1e2dc7_develop</judo-psm-generator-sdk-core-version>
+        <judo-tatami-jsl-version>1.1.4.20230607_040731_6747bbf3_develop</judo-tatami-jsl-version>
+        <judo-runtime-core-version>1.0.6.20230606_125352_f2806e04_develop</judo-runtime-core-version>
+        <judo-psm-generator-sdk-core-version>1.0.0.20230612_162055_53b22a7d_feature_JNG_4800_fix_jsl_model_name_with_java_keyword_throws_error</judo-psm-generator-sdk-core-version>
```
Comment (bencegelei) [inline pom.xml]: Version downgrading

File: judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/ModelNameWithJavaKeyword.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    @Override
+    public Module getModelDaoModule() {
+        return new AbstractDaoModules();
+    }
+
+    @Override
+    public String getModelName() {
+        return "Abstract";
+    }
```
Comment (bencegelei) [inline judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/ModelNameWithJavaKeyword.java:29]: Please add least one assert to test

**Review by bencegelei — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (Madmat8) [reply pom.xml]: [2cbc104](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/197/commits/2cbc1046eae050a4eae3c5fddb4d855da72067dc)

File: judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/ModelNameWithJavaKeyword.java (hunk 2 above)
Comment (Madmat8) [reply judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/ModelNameWithJavaKeyword.java:29]: [e3c4d15](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/197/commits/e3c4d1573c75fd07f7f0e2835350da6ed81bea7f)

**Review by gaborflorian — APPROVED** [review]

**Review by bencegelei — APPROVED** [review]

**Pull request merged**